### ISSUE TITLE
WebNN's WPT test seems blocking the CL submission of Model Loader API

### ISSUE THREAD

**Issue by yuhonglin** [webmachinelearning/model-loader#31]
Error message: https://ci.chromium.org/ui/p/chromium/builders/try/win10_chromium_x64_rel_ng/b8818945326557480433/test-results?q=ExactID%3Aninja%3A%2F%2F%3Ablink_web_tests%2Fexternal%2Fwpt%2Fwebnn%2Fidlharness.https.any.html+VHash%3A7a1d2119a1573123

The test: https://github.com/web-platform-tests/wpt/tree/master/webnn

**Comment by yuhonglin** [comment]
@huningxin Can we disable that test temporarily?

**Comment by yuhonglin** [comment]
No worries for now. It seems we can disable it in chrome: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/TestExpectations;l=1?q=TestExpectations&sq=&ss=chromium

**Comment by huningxin** [comment]
good to know it won't block model loader. /cc @BruceDai who is working on WebNN WPT.

**Comment by yuhonglin** [comment]
Never mind. We will update the "expectation" of the test output to make it pass the chromium cq. It seems a common practice to do so.

**Comment by BruceDai** [comment]
@yuhonglin Since there're common ML and MLContext interfaces in both WebNN and Model loader API, this would make some WebNN IDL tests pass in your CL, glad to see you found the solution.
BTW, we think you could also add IDL tests of Model Loader API into WPT, what's your plan for it?

**Comment by yuhonglin** [comment]
Yes, "adding Model loader test to WPT" is on our todo list but is not of high priority. We currently lack bandwidth so, many things are moving slower (including my replies on Github :) ) than expected. Sorry.
